pilot/main.py

```diff
@@ -1,11 +1,15 @@
 # main.py
-from __future__ import print_function, unicode_literals
 import builtins
 import os
 
 import sys
 import traceback
-from dotenv import load_dotenv
+
+try:
+    from dotenv import load_dotenv
+except ImportError:
+    raise RuntimeError('Python environment for GPT Pilot is not completely set up: required package "python-dotenv" is missing.') from None
```

Comment:
Again not related but crept in somehow. This seems like the first place were we're loading Python packages that are not built in, so if the virtual environment is not correctly set up, this raises an error.

This change will still raise an error, but a bit nicer one, so that people have better idea of what's going wrong.